### DISCUSSION TITLE
Is a phising website?

### DISCUSSION THREAD

**Discussion by ivanbeltrame** [community/community#40302] [New to GitHub] [answered]
I found this website: [Link Redacted due to phishing concerns]
Is it a phising website?

**Comment by mostafasoufi** [comment]
It seems so.

**Reply by ivanbeltrame** [reply]
Is there anything I can do? Can I report it directly to GitHub?

**Reply by mostafasoufi** [reply]
Yeah, just report it.

**Comment by airtower-luna** [comment, marked as answer]
The contact form for support is here: https://support.github.com/request This isn't about content on GitHub, so I suppose it falls under "other support", but I'm not sure. :sweat_smile: 

You might also want to report it to Google, their ["Safe Browsing" blocklist](https://safebrowsing.google.com/) is used by various browsers, which then block the site: https://safebrowsing.google.com/safebrowsing/report_phish/
